conftest.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import patch
+
+import pytest
+
+from sso.utils import SSOUser
+
+
+@pytest.fixture
+def sso_user():
```

Comment:
If this is used as a local fixture elsewhere, it will need to be replaced with the global fixture in a future PR